docs/install_guides/installing-openvino-overview.md

```diff
@@ -6,7 +6,8 @@
    :maxdepth: 3
    :hidden:
    
-   OpenVINO Runtime <openvino_docs_install_guides_install_runtime>
+   OpenVINO Runtime 2022.1 <openvino_docs_install_guides_install_runtime>
+   OpenVINO Runtime 2022.1.1 <openvino_docs_install_guides_install_runtime_2022_1_1>
```

Comment:
Do we really want to have 2 installation guides depending on the version?

Originally, I supposed that we can select documentation version like latest (2022.1 since we want most of the users use 2022.1) and 2022.1.1 for special users like MSFT. How do you think? @helena-intel @ryanloney